tripleblindmarket/private-kit
# Add automated lokalise import

#### Description:

This commit adds a GitHub action to pull the latest lokalise data and generate a pull request with it. It is scheduled to run every
4 hours.

#### Linked issues:

Fixes SAF-212

#### Screenshots:

<none>

#### How to test:

Once merged, we should see the PR coming/being updated every 4 hours, if any lokalise data has been updated. Alternatively, we could test it by adding "on: \n workflow_dispatch:" to the GitHub workflow definition, so we can run it on demand instead of a fixed schedule.
